src/components/TokenGeneralInfo.js

```diff
@@ -31,18 +34,26 @@ const mapStateToProps = (state) => {
  */
 class TokenGeneralInfo extends React.Component {
   /**
-   * errorMessage {String} Message to show in case of error getting token info
-   * totalSupply {number} Token total supply
-   * canMint {boolean} If this token can still be minted
-   * canMelt {boolean} If this token can still be melted
-   * transactionsCount {number} Total number of transactions of this token
+   * @property {string} errorMessage Message to show in case of error getting token info
+   * @property {number} totalSupply Token total supply
+   * @property {boolean} canMint If this token can still be minted
+   * @property {boolean} canMelt If this token can still be melted
+   * @property {number} transactionsCount Total number of transactions of this token
+   * @property {boolean} alwaysShow Indicates if this token is always shown despite having zero bal.
+   * @property {{title:string,body:string,handleYes:function}} confirmData Confirm modal settings
    */
   state = {
     errorMessage: '',
     totalSupply: 0,
     canMint: null,
     canMelt: null,
     transactionsCount: 0,
+    alwaysShow: false,
+    confirmData: {
+      title: '',
+      body: '',
+      handleYes: () => {}
```

Comment:
```suggestion
      handleYes: () => {},
```

Reply:
Fixed on 04b6f48c6579b568b9caf55233c29dca14e31ecb .
I thought we already had a _linter_ running on this application. I'll open an issue to help with that.